Jenkinsfile

```diff
@@ -399,8 +399,7 @@ def runScript(Map params = [:]){
   sh "mkdir ${env.PIP_CACHE}"
   unstash 'source'
   dir("${BASE_DIR}"){
-    retry(2){
-      sleep randomNumber(min:10, max: 30)
+    retryWithSleep(retries: 2, seconds: 5, backoff: true) {
```

Comment:
unrelated with this PR, but using the more up to date step to do retry with a sleep